pskb_website/static/js/editor_utils.js

```diff
@@ -317,6 +317,7 @@ function save(sha, path, secondary_repo, action_url) {
     var stacks = document.createElement("input");
     stacks.name = "stacks";
     stacks.value = stacks_select.value;
+    stacks.required = "required";
```

Comment:
I was just trying to improve the form, since I saw in the frontend we discard empty values for the stack.

But we can ignore this piece of code since I replaced this with the new version using Jquery.Ajax